pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConsumerConfigurationData.java

```diff
@@ -437,4 +437,12 @@ public ConsumerConfigurationData<T> clone() {
             throw new RuntimeException("Failed to clone ConsumerConfigurationData");
         }
     }
+
+    /**
+     * @deprecated Using {@link #getReplicateSubscriptionState()} instead.
+     */
+    @Deprecated
```

Comment:
Is `@JsonIgnore` needed on this method?

Reply:
@lhotari Yes, `@JsonIgnore` has been added.

Reply:
`@JsonProperty(access = JsonProperty.Access.READ_WRITE)` is required, this will ensure that the ·replicateSubscriptionState· field is properly handled during JSON serialization and deserialization